src/server/routes/consents.ts

```diff
@@ -365,6 +367,10 @@ router.post(
         state.queryParams,
       );
 
+      setEncryptedStateCookie(res, {
+        cmpConsentedState: _cmpConsentedState,
```

Comment:
Instead what you could do is to use `isStringBoolean` here to covert this string into a boolean

```suggestion
        cmpConsentedState: isStringBoolean(_cmpConsentedState),
```

Reply:
Thanks - I just realised this! will push a fix momentarily.

Reply:
I will see if I can amend the form to submit a proper boolean instead of a "boolean"